vcpkg-configuration.json

```diff
@@ -0,0 +1,14 @@
+{
+  "default-registry": {
+    "kind": "git",
+    "baseline": "6f1ddd6b6878e7e66fcc35c65ba1d8feec2e01f8",
+    "repository": "https://github.com/microsoft/vcpkg"
+  },
+  "registries": [
+    {
+      "kind": "artifact",
+      "location": "https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",
+      "name": "microsoft"
+    }
+  ]
+}
```

Comment:
two questions:
1. why do we need this file at all? AFAIK when the user sets up vcpkg they already clone the registry
2. I'm not sure what "vcpkg-ce" is and it doesn't look like we need it

Reply:
1. We may not need it, I'll see if it's an autogenerated file that can be ignored. Note that the new docs recommend using the `vcpkg` that comes built-in with VS, and even then vcpkg is used only indirectly through CMake. This simplifies the workflow a good amount IMO as a new dev.

2. `vcpkg-ce-catalog` looks like the registry used for installing artifacts. If we do need this file, I'll see what happens when this `registries` array is set to empty.

Reply:
Ugh, ran out of disk space, gonna have to try again